pages/colonydetail/[id].js

```diff
@@ -11,8 +12,10 @@ export default function ColonyDetailsPage({ colonies }) {
 
   return (
     <>
-      <Card text={currentColony?.colonyName} />
-      <ColonyQuestions />
+      <StyledList>
+        <Card text={currentColony?.colonyName} isBold={true} />
+        <ColonyQuestions currentColony={currentColony} />
```

Comment:
I wanted to have the list so that everything is centered, but I changed it. 

And again, about the headline not as a card, I might think about that at a later point, but for now I like that way.